packages/react-native/React/Base/RCTBundleURLProvider.mm

```diff
@@ -308,6 +308,7 @@ + (NSURL *)jsBundleURLForBundleRoot:(NSString *)bundleRoot
   BOOL lazy = enableDev;
   NSArray<NSURLQueryItem *> *queryItems = @[
     [[NSURLQueryItem alloc] initWithName:@"platform" value:RCTPlatformName],
+    [[NSURLQueryItem alloc] initWithName:@"resolver.variant" value:RCTPlatformVariant],
```

Comment:
I think we should also add `#if TARGET_OS_VISION` here

Reply:
Ok. We'll think about making it more generic when upstreaming to the core